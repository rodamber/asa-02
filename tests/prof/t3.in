6 8
1
1 2 100
2 3 100
1 3 198
3 4 100
1 4 280
4 5 100
5 1 -350
6 1 -100

