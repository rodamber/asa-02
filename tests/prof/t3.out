0
100
198
280
380
U
